docs/pages/web/textarea.js

```diff
@@ -635,6 +638,36 @@ function Example(props) {
     </Box>
   );
 }
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="With rows"
+          description={`
+  You can use the rows property to set the max height of the Textarea. It will begin to overflow after the rows limit has been reached`}
+        >
+          <MainSection.Card
+            defaultCode={`          
+function Example(props) {
+
+const [value, setValue] = React.useState('');
+const [rows, setRows] = React.useState(2);
+
+return (
+  <Box width="100%">
+    <Box marginBottom={2}>
+      <NumberField label="Number of Rows" onChange={({value})=>{setRows(value)}} value={rows}/>
```

Comment:
![Screen Shot 2023-04-25 at 6 04 49 PM](https://user-images.githubusercontent.com/12059539/234439927-3c37c26e-30f2-4a94-99c8-5d3127ceaba7.png)

The UI looks a bit goofy having this NumberField the full width of the TextArea. Consider limiting this width to something that would accommodate two digits or so.